test/tsconfig.json

```diff
@@ -11,6 +11,9 @@
         "strictNullChecks": false,
         "types": ["vitest/globals"],
         "module": "Node16",
-        "moduleResolution": "Node16"
+        "moduleResolution": "Bundler",
```

Comment:
why is that?

Reply:
See https://www.typescriptlang.org/tsconfig#moduleResolution

TL;DR: we don't want TS to require the extension when importing relative paths like we do in some tests. But we *do* want it to resolve the import

Reply:
i'd be actually fine with adding the extensions, since we want to move to ESM at some point anyway. but i guess it doesnt really matter for tests.